tests/io/test_structure.py

```diff
@@ -1443,21 +1444,39 @@ def test_orifice_parses_flowdirection_case_insensitive(self, input, expected):
 
         assert structure.allowedflowdir == expected
 
-    def _create_orifice_values(self) -> dict:
+    def test_optional_fields_have_correct_defaults(self):
+        structure = Orifice(**self._create_required_orifice_values())
+
+        assert structure.allowedflowdir == FlowDirection.both
+        assert structure.crestwidth == None
+        assert structure.uselimitflowpos == False
+        assert structure.limitflowpos == None
+        assert structure.uselimitflowneg == False
+        assert structure.limitflowneg == None
+
+    def _create_required_orifice_values(self) -> dict:
         return dict(
             id="structure_id",
             name="structure_name",
             type="orifice",
             branchid="branch_id",
             chainage="1.23",
-            allowedflowdir="positive",
             crestlevel="2.34",
-            crestwidth="3.45",
             gateloweredgelevel="4.56",
             corrcoeff="5.67",
             usevelocityheight="true",
+        )
+
+    def _create_orifice_values(self) -> dict:
+        optional_values = dict(
```

Comment:
confusing name ```optional_values```, because in the lines below the required ones are also added.

Reply:
Have changed it to orifice_values